### PR TITLE
Update readme

### LINKED ISSUE #105: Update readme
The link to Substitution Cipher is broken in readme and newer inclusions are not present.

### PR DESCRIPTION
**Pull request by ivaibhavjindal**
Please review.
Closes #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,28 +12,57 @@ This is a library of various algorithms that are used in competitive programming
...
 [Selection Sort](Algorithms/Sorting/selection_sort.cpp)
 
+### Stacks
+[Implementation](Algorithms/Stacks/Stack%20in%20Array.cpp) <br/>
+[Infix to Postfix](Algorithms/Stack/InfixToPostfix.java) <br/>
+[Postfix](Algorithms/Stack/EvaluatePostfixExpression.txt)
+
 ### Strings
-[Substitution_Cipher](https://github.com/NJACKWinterOfCode/nwoc_algorithms/tree/master/Algorithms/Substitution_Cipher)
+[Substitution Cipher](https://github.com/NJACKWinterOfCode/nwoc_algorithms/tree/master/Algorithms/Substitution_Cipher)
```
Comment (Bhavnaharitsa) [inline README.md:59]: No change dome over here

File: README.md [hunk 2]
```diff
@@ -12,28 +12,57 @@ This is a library of various algorithms that are used in competitive programming
...
 
 ### String Hashing
-[Count_Substring](Algorithms/String_Hashing/count_substr.cpp) <br/>
+[Count Substring](Algorithms/String_Hashing/count_substr.cpp) <br/>
 [Rabin Karp Algorithm](Algorithms/String_Hashing/rabin_karp.cpp)<br/>
```
Comment (Bhavnaharitsa) [inline README.md:63]: Same

File: README.md (hunk 1 above)
Comment (ivaibhavjindal) [reply README.md:59]: What are the changes to be done?

Comment (Bhavnaharitsa) [reply README.md:59]: No i mean the commit does not contain change

**Comment by Bhavnaharitsa** [comment]
Conflicts

**Comment by ivaibhavjindal** [comment]
Conflicts have been resolved. Newer Algos have been included in lexicographic order and Headings fixed. Also the current Readme has BellmanFord Algorithm mentioned twice. This fixes that too.

**Pull request merged**